portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentUI.testcase

```diff
@@ -231,4 +231,66 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-131307. View the order of web content actions."
+	@priority = "5"
+	test ViewOrderOfWebContentActions {
+		task ("Add a published web content based on Basic Web Content") {
+			JSONWebcontent.addWebContent(
+				content = "WC WebContent Content",
+				groupName = "Test Site Name",
+				title = "Published WebContent Title");
+		}
+
+		task ("Add a draft web content based on Basic Web Content") {
+			JSONWebcontent.addWebContent(
+				content = "WC WebContent Content",
+				groupName = "Test Site Name",
+				title = "Draft WebContent Title",
+				workflowAction = "DRAFT");
+		}
+
+		task ("Add an expired web content based on Basic Web Content") {
+			JSONWebcontent.addWebContent(
+				content = "WC WebContent Content",
+				groupName = "Test Site Name",
+				title = "Expired WebContent Title");
+
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+			WebContent.expireCP(webContentTitleList = "Expired WebContent Title");
```

Comment:
We should find a way to do this with JSON

Reply:
I tried. It seems like json ws doesn’t support that. Anyway, I will confirm with our devs.